resources/application.rb

```diff
@@ -0,0 +1,80 @@
+#
+# Author:: Madan Kapoor (<madankapoor10@gmail.com>)
+# Cookbook:: webpi
+# Resource:: application
+#
+# Copyright:: 2011-2017, Chef Software, Inc.
```

Comment:
This should be copyright yourself not us. You wrote it and you retain copyright on the work.